gtsfm/frontend/inlier_support_processor.py

```diff
@@ -60,6 +59,20 @@ def run(
             v_corr_idxs: empty (0,2) array if insufficient support
             two_view_report: two-view estimation report, or None if insufficient support
         """
+        # i2Ri1 is None, and i2Ui1 is None upon failure.
+        failure_result = (
+            None,
+            None,
+            np.array([], dtype=np.uint64),
+            TwoViewEstimationReport(v_corr_idxs=v_corr_idxs, num_inliers_est_model=0),
+        )
+
+        # if (two_view_report.R_error_deg is not None) and (two_view_report.R_error_deg > 5):
+        #     return failure_result
+
+        # if (two_view_report.U_error_deg is not None) and (two_view_report.U_error_deg > 5):
+        #     return failure_result
+
```

Comment:
Why are we removing this?

Reply:
Oh this code was actually just for debugging (uses GT oracle error). Removed now. ISP is unchanged.